README.md

```diff
@@ -739,12 +753,9 @@ which will be included in Emacs 27. It is basically a variation of
 Icomplete that behaves more like Ido. As such, Fido mode does not
 offer solutions to the problems outlined in the above sections.
 
-To be transparent, there are a few standard Emacs features which are
-not implemented in Selectrum (mostly because I was unaware they
-existed) but which do work in Icomplete: for example, some of the
-special `completion-*` user options such as
-`completion-ignored-extensions`. I do not see any design reason these
-features cannot all be incorporated into Selectrum eventually.
+On the upside, Icomplete is probably the most API compliant continious
+completion UI available. See the Caveats section for details about
+which features are not supported in Selectrum, yet.
```

Comment:
Ah, you already did what I had in mind!

Reply:
continious - typo

> See the Caveats section for details about
which features are not supported in Selectrum, yet.

I would formulate it a bit differently. "On the upside, Icomplete is the most API compliant completion UI available. Selectrum strives hard to achieve full compliance as well. There are only very few unsupported edge cases left, see caveats..."

Reply:
There is still the default UI itself, that is why I wanted to bring in continuous completion somehow. I rephrased it using some parts of you suggestion, take a look.

Reply:
Sounds good!